SADXModManager/MainForm.cs

```diff
@@ -308,6 +310,14 @@ private void MainForm_Shown(object sender, EventArgs e)
             if (CheckForUpdates())
                 return;
 
+            if (File.Exists("d3d8m.dll"))
+                checkBoxEnableD3D9.Enabled = true;
+
+            if (File.Exists("d3d8.dll"))
+            {
+                checkBoxEnableD3D9.Checked = true;
+            }
```

Comment:
Nitpick, but this can be simplified like so:
```suggestion
            checkBoxEnableD3D9.Checked = File.Exists("d3d8.dll");
```
I am not against the current form for debugging purposes, however.

Reply:
Also, all the instances of the string "d3d8.dll" ought to be replaced with a constant string, like for example, a const field:

```cs
private const string d3d8to9InstalledDLLName = "d3d8.dll";
```